app/src/main/res/layout/widechat_search_layout.xml

```diff
@@ -9,25 +9,50 @@
     android:paddingEnd="8dp"
     android:paddingStart="0dp">
 
-    <com.facebook.drawee.view.SimpleDraweeView
-        android:id="@+id/profile_image_avatar"
+
+
+    <RelativeLayout
+        android:id="@+id/rl_image_avatar"
         android:layout_width="48dp"
         android:layout_height="48dp"
-        android:layout_marginRight="24dp"
-        android:foregroundGravity="center"
-        android:background="@drawable/widechat_circle_shadow"
-        app:roundedCornerRadius="3dp"
-        fresco:actualImageScaleType="centerCrop"
-        fresco:roundAsCircle="true"
-        fresco:roundingBorderColor="@color/colorWhite"
-        fresco:roundingBorderPadding="2dp"
-        fresco:roundingBorderWidth="2dp" />
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginRight="16dp"
+        android:layout_margin="10dp">
+
+        <com.facebook.drawee.view.SimpleDraweeView
```

Comment:
It now looks like the profile image avatar is not centered between the parent start and the search box.  Can you adjust that please?

Reply:
Done

Reply:
This still looks not centered to me, when previously it was.... do you see that?

![image](https://user-images.githubusercontent.com/28864717/50293883-9d01bc00-0442-11e9-996e-6f7fb94c1fb3.png)